.zuul.playbooks/playbooks/wheel/roles/build-wheel-manylinux/files/build-wheels.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash -x
+# Compile wheels
+
+cd /io
+mkdir wheelhouse.final
+
+for PYBIN in /opt/python/*/bin; do
+
+    "${PYBIN}"/python -m virtualenv .venv
+
+    .venv/bin/pip install cffi six ipaddress "enum34; python_version < '3'"
+
+    REGEX="cp3([0-9])*"
+
+    #if [[ "${PYBIN}" =~ $REGEX ]]; then
```

Comment:
Let's uncomment this since we're going to generate abi3 wheels (and only abi3 wheels)

Reply:
Done, converted to use bdist_wheel